tests/MongoDB.Driver.Core.TestHelpers/CoreExceptionHelper.cs

```diff
@@ -117,5 +118,17 @@ public static MongoCommandException CreateMongoWriteConcernException(BsonDocumen
 
             return writeConcernException;
         }
+
+        public static SocketException CreateSocketException(string errorType)
+        {
+            switch (errorType)
+            {
+                case "timedout":
+                    return new SocketException((int)SocketError.TimedOut);
+                case "networkunreachable":
+                    return new SocketException((int)SocketError.NetworkUnreachable);
+                default: throw new ArgumentException("Unknown error type.");
```

Comment:
`throw` should be on new line to match other cases

Reply:
Done!